codegen/type_converter.go

```diff
@@ -458,9 +489,10 @@ func (c *TypeConverter) genConverterForMap(
 			fromFieldType.ValueSpec,
 			valID,
 			keyPrefix+pascalCase(toField.Name)+"["+keyID+"]",
-			strings.TrimPrefix(fromIdentifier, "in.")+"["+keyID+"]",
+			strings.TrimPrefix(strings.TrimPrefix(fromIdentifier, "in."), "inOriginal.")+"["+keyID+"]",
```

Comment:
mm ... you can do better here :) `TrimPrefix` twice seems too much implication, what's the unambiguous way of saying: 

I want to get the real arg name without all the prefixes I might have added

Reply:
Hmm okay I'll add a utility method to do this